docs/news/posts/2023-09-05-announcing-discourse.md

```diff
@@ -0,0 +1,32 @@
+---
+blogpost: true
+category: News
+tags: communication
+date: 2023-09-05
+---
+
+# Announcing a new communication platform
+
+Today we announce the launch of a new communication channel for AiiDA:
+
+[https://aiida.discourse.group/](https://aiida.discourse.group/)
+
+This new platform will replace the various other solutions that are currently in use: the [mailing list](http://groups.google.com/group/aiidausers), the [Slack workspace](https://aiidateam.slack.com) and [GitHub discussions](https://github.com/orgs/aiidateam/discussions).
+All of these suffer from various weaknesses, such as little to no support for code formatting, not being publicly accessible and generally being intimidating to use for (new) users.
+In addition, having multiple platforms caused valuable information to become dispersed and difficult to find.
+
+The new forum, based on the [Discourse open source discussion platform](https://www.discourse.org/), solves these issues by providing a single public space for discussions between AiiDA users and developers, and for users to get support:
+
+All data is publicly accessible without the need to create an account.
+This makes discussions discoverable by search engines.
+It is easy for new users to contribute to discussions or request support: anyone can sign up (or use a Github account).
```

Comment:
Might also want to make it possible to use a Google account, looking into that.

Reply:
I propose we merge this. If you manage to enable it, you can update the news item.